packages/panzer/mini-em/example/BlockPrec/solverCG.xml

```diff
@@ -43,7 +43,7 @@
               <ParameterList name="Pseudo Block CG">
                 <Parameter name="Convergence Tolerance" type="double" value="1.0e-9"/>
                 <Parameter name="Orthogonalization" type="string" value="ICGS"/>
-                <Parameter name="Maximum Iterations" type="int" value="100"/>
+                <Parameter name="Maximum Iterations" type="int" value="2000"/>
```

Comment:
With bumping the iterations significantly, does the test runtime have a corresponding increase?

Reply:
It does, but I also reduced the test size.